superset/assets/src/setup/setupApp.js

```diff
@@ -20,6 +20,16 @@
 import $ from 'jquery';
 import { SupersetClient } from '@superset-ui/connection';
 import getClientErrorObject from '../utils/getClientErrorObject';
+import moment from 'moment';
+import localStorage from 'local-storage';
```

Comment:
Why is this needed instead of using the native `localStorage`?

Reply:
You're right, it's not needed in this case - I'm just used to using it.